locales/messages.de.json

```
@@ -51,23 +51,23 @@
   "NoLocalizationForMessages": "Keine Lokalisierung für die folgenden Nachrichten:",
-  "ProcessorArchitectureMalformed": "Fehler beim Analysieren von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
+  "ProcessorArchitectureMalformed": "Fehler beim parsen von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
   "ProcessorArchitectureMissing": "Die erforderliche Umgebungsvariable %PROCESSOR_ARCHITECTURE% fehlt.",
-  "ProcessorArchitectureW6432Malformed": "Fehler beim Analysieren von %PROCESSOR_ARCHITEW6432% ({arch}) als eine gültige CPU-Architektur. Fallback auf %PROCESSOR_ARCHITECTURE% wird durchgeführt.",
+  "ProcessorArchitectureW6432Malformed": "Fehler beim parsen von %PROCESSOR_ARCHITEW6432% ({arch}) als eine gültige CPU-Architektur. Fallback auf %PROCESSOR_ARCHITECTURE% wird durchgeführt.",
   "SeeURL": "Weitere Informationen finden Sie unter {url}.",
   "SuggestNewVersionScheme": "Verwenden Sie das Versionsschema „{new_scheme}“ anstelle von „{old_scheme}“ in Port „{package_name}“. \nVerwenden Sie „--{option}“, um diese Überprüfung zu deaktivieren.",
-  "UnsupportedSystemName": "Fehler: VCPKG_CMAKE_SYSTEM_NAME „{system_name}“ konnte keiner vcvarsall-Plattform zugeordnet werden. Unterstützte Systemnamen sind „,“, „Windows“ und „WindowsStore“.",
-  "UnsupportedToolchain": "Fehler: Im Tripel {triplet}: Es wurde keine gültige Toolkettenkombination gefunden.\n    Die angeforderte Zielarchitektur war {arch}\n    Die ausgewählte Visual Studio-Instanz befindet sich unter {path}\n    Die verfügbaren Toolkettenkombinationen sind {list}\n",
+  "UnsupportedSystemName": "Fehler: VCPKG_CMAKE_SYSTEM_NAME „{system_name}“ konnte keiner vcvarsall-Plattform zugeordnet werden. Unterstützte Systemnamen sind „“, „Windows“ und „WindowsStore“.",
+  "UnsupportedToolchain": "Fehler: Im Triplett {triplet}: Es wurde keine gültige Toolset-Kombination gefunden.\n    Die angeforderte Zielarchitektur war {arch}\n    Die ausgewählte Visual Studio-Instanz befindet sich unter {path}\n    Die verfügbaren Toolset-Kombinationen sind {list}\n",
   "UsingManifestAt": "Die Manifestdatei unter {path} wird verwendet.",
   "VSExaminedInstances": "Die folgenden Visual Studio-Instanzen wurden berücksichtigt:",
   "VSExaminedPaths": "Die folgenden Pfade wurden für Visual Studio-Instanzen untersucht:",
   "VSNoInstances": "Es konnte keine vollständige Visual Studio-Instanz gefunden werden.",
-  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Verteilung verfügt nicht über eine Instanz im klassischen Modus.",
+  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Verteilung(ROFL) verfügt nicht über eine Instanz im klassischen Modus.",
```

```suggestion
  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Installation verfügt nicht über den klassischen Modus.",
```

I don't quite get how `distribution` works together with `instance` here. 
Maybe the English text needs rewording for that first.

---

very likely.

---

(internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565252